### PR TITLE
Update release process doc

### PR DESCRIPTION
**Pull request by zymap**
---

*Motivation*

When we release the Pulsarctl, we need to check
the doc link and work with Growth to announce the
Pulsarctl released.

### CODE REVIEW

**Comment by sijie** [comment]
@Anonymitaet please help review this pull request.

File: docs/en/release_process.md [hunk 1]
```diff
@@ -6,13 +6,24 @@ This guide illustrates how to perform a release for pulsarctl.
 
 The steps for releasing are as follows:
 
-1. Create the release branch
-2. Promote the release
-3. Write a release note
+1. Prepare for releasing
```
Comment (Anonymitaet) [inline docs/en/release_process.md]:
```suggestion
1. Prepare for a release
```

File: docs/en/release_process.md [hunk 2]
```diff
@@ -6,13 +6,24 @@ This guide illustrates how to perform a release for pulsarctl.
...
+2. Create the release branch
+3. Promote the release
+4. Write a release note
+5. Check the release
+6. Announce the release
 
 ## Steps in detail
 
-#### 1. Create the release branch
+#### 1. Prepare for releasing
```
Comment (Anonymitaet) [inline docs/en/release_process.md]:
```suggestion
1. Prepare for release.
```

File: docs/en/release_process.md [hunk 3]
```diff
@@ -6,13 +6,24 @@ This guide illustrates how to perform a release for pulsarctl.
...
+
+Create a new milestone and move those pull requests that can not
```
Comment (Anonymitaet) [inline docs/en/release_process.md]:
```suggestion
Create a new milestone and move the pull requests that can not
```

File: docs/en/release_process.md [hunk 4]
```diff
@@ -6,13 +6,24 @@ This guide illustrates how to perform a release for pulsarctl.
...
+publish in this release to the new milestone.
```
Comment (Anonymitaet) [inline docs/en/release_process.md]:
```suggestion
be published in this release to the new milestone.
```

File: docs/en/release_process.md [hunk 5]
```diff
@@ -6,13 +6,24 @@ This guide illustrates how to perform a release for pulsarctl.
...
+
+Update the VERSION file and stable.txt file to the new version.
```
Comment (Anonymitaet) [inline docs/en/release_process.md]:
```suggestion
Add the information of the new release to the `VERSION file` and `stable.txt file` and send a PR for requesting the changes.
```

File: docs/en/release_process.md [hunk 6]
```diff
@@ -6,13 +6,24 @@ This guide illustrates how to perform a release for pulsarctl.
...
+And send a pull request for updating the files.
```
Comment (Anonymitaet) [inline docs/en/release_process.md]:
```suggestion

```

File: docs/en/release_process.md [hunk 7]
```diff
@@ -6,13 +6,24 @@ This guide illustrates how to perform a release for pulsarctl.
...
+
+#### 2. Create the release branch
```
Comment (Anonymitaet) [inline docs/en/release_process.md]:
```suggestion
2. Create the release branch
```

File: docs/en/release_process.md [hunk 8]
```diff
@@ -36,20 +47,18 @@ $ cd pulsarctl
 $ git checkout -b branch-0.1.X origin/master
 ```
 
-#### 2. Create and push the release.
+#### 3. Create and push the release
```
Comment (Anonymitaet) [inline docs/en/release_process.md]:
```suggestion
3. Push the release
```

File: docs/en/release_process.md [hunk 9]
```diff
@@ -6,13 +6,24 @@ This guide illustrates how to perform a release for pulsarctl.
 
 The steps for releasing are as follows:
 
-1. Create the release branch
-2. Promote the release
-3. Write a release note
+1. Prepare for releasing
+2. Create the release branch
+3. Promote the release
```
Comment (Anonymitaet) [inline docs/en/release_process.md]:
```suggestion
3. Push the release
```

File: docs/en/release_process.md [hunk 10]
```diff
@@ -36,20 +47,18 @@ $ cd pulsarctl
...
-# Update VERSION file
-$ echo v0.1.X > VERSION
 
 # Push both the branch and the tag to Github repo
 $ git push origin branch-0.1.X
 $ git push origin v0.1.X
 ```
 
-#### 3. Publish a release note
+#### 4. Publish a release note
```
Comment (Anonymitaet) [inline docs/en/release_process.md]:
```suggestion
4. Publish a release note
```

File: docs/en/release_process.md [hunk 11]
```diff
@@ -58,3 +67,21 @@ In the released item, add the list of the most important changes that happened i
 Update the release draft at [the release homepage of pulsarctl](https://github.com/streamnative/pulsarctl/releases)
 
 Then publish the release draft, the binary and commands doc will auto publish to that release
```
Comment (Anonymitaet) [inline docs/en/release_process.md]:
```suggestion
Then the release draft, the binary, and the command doc will be published to that release automatically.
```

File: docs/en/release_process.md [hunk 12]
```diff
@@ -58,3 +67,21 @@ In the released item, add the list of the most important changes that happened i
...
+
+#### 5. Check the release
```
Comment (Anonymitaet) [inline docs/en/release_process.md]:
```suggestion
5. Check the release
```

File: docs/en/release_process.md [hunk 13]
```diff
@@ -58,3 +67,21 @@ In the released item, add the list of the most important changes that happened i
...
+
+View the doc site https://streamnative.io/docs/pulsarctl/vx.y.z/ to
```
Comment (Anonymitaet) [inline docs/en/release_process.md]:
```suggestion
(1) Visit [Puslarctl website](https://streamnative.io/docs/pulsarctl/vx.y.z/) to
```

File: docs/en/release_process.md [hunk 14]
```diff
@@ -58,3 +67,21 @@ In the released item, add the list of the most important changes that happened i
...
+check the docs have been published successfully.
+
```
Comment (Anonymitaet) [inline docs/en/release_process.md]:
```suggestion
`vx.y.z` is the version of the release. For example, https://streamnative.io/docs/pulsarctl/v0.1.0/.
```

File: docs/en/release_process.md [hunk 15]
```diff
@@ -58,3 +67,21 @@ In the released item, add the list of the most important changes that happened i
...
+Use the `install.sh` to install the Pulsarctl and make sure the
```
Comment (Anonymitaet) [inline docs/en/release_process.md]:
```suggestion
(2) Use the install command (`sh -c "$(curl -fsSL https://raw.githubusercontent.com/streamnative/pulsarctl/master/install.sh)"`) to install the Pulsarctl and make sure the
```

File: docs/en/release_process.md [hunk 16]
```diff
@@ -58,3 +67,21 @@ In the released item, add the list of the most important changes that happened i
...
+version of the download Pulsarctl is right.
```
Comment (Anonymitaet) [inline docs/en/release_process.md]:
```suggestion
version of the downloaded Pulsarctl is right.
```

File: docs/en/release_process.md [hunk 17]
```diff
@@ -58,3 +67,21 @@ In the released item, add the list of the most important changes that happened i
...
+
+Add the latest version to the `Available Release` in `README.md`.
```
Comment (Anonymitaet) [inline docs/en/release_process.md]:
```suggestion
(3) Add the latest version to the `Available Release` in `README.md`.
```

File: docs/en/release_process.md [hunk 18]
```diff
@@ -58,3 +67,21 @@ In the released item, add the list of the most important changes that happened i
...
+
+Delete the previous milestone.
```
Comment (Anonymitaet) [inline docs/en/release_process.md]:
```suggestion
(4) Delete the previous milestone.
```

File: docs/en/release_process.md [hunk 19]
```diff
@@ -58,3 +67,21 @@ In the released item, add the list of the most important changes that happened i
...
+
+#### 6. Announce the release
```
Comment (Anonymitaet) [inline docs/en/release_process.md]:
```suggestion
6. Announce the release
```

File: docs/en/release_process.md [hunk 20]
```diff
@@ -58,3 +67,21 @@ In the released item, add the list of the most important changes that happened i
...
+
+After publishing and checking the release. Please work
```
Comment (Anonymitaet) [inline docs/en/release_process.md]:
```suggestion
After publishing and checking the release, work
```

File: docs/en/release_process.md [hunk 21]
```diff
@@ -58,3 +67,21 @@ In the released item, add the list of the most important changes that happened i
...
+with Growth to announce we are release the new version
```
Comment (Anonymitaet) [inline docs/en/release_process.md]:
```suggestion
with Growth to announce that a new version of Pulsarctl is released.
```

File: docs/en/release_process.md [hunk 22]
```diff
@@ -58,3 +67,21 @@ In the released item, add the list of the most important changes that happened i
...
+of the Pulsarctl.
```
Comment (Anonymitaet) [inline docs/en/release_process.md]:
```suggestion

```

**Comment by zymap** [comment]
@sijie PTAL. Thanks.

If there is nothing need to change. I will merge this PR.

File: docs/en/release_process.md [hunk 23]
```diff
@@ -25,36 +37,58 @@ Eg: When creating `v0.1.1` release, will be creating
...
+(1) Visit the Pulsarctl website https://streamnative.io/docs/pulsarctl/vx.y.z/ to
+check the docs have been published successfully. `vx.y.z` is the version of the
+release. For example, https://streamnative.io/docs/publisctl/v0.1.0/.
+
+(2) Use the install command `sh -c "$(curl -fsSL https://raw.githubusercontent.com/streamnative/pulsarctl/master/install.sh)"`
+to install the Pulsarctl and make sure the version of the downloaded Pulsarctl is right.
+
+(3) Add the latest version to the `Available Release` in `README.md`.
+
+(4) Delete the previous milestone.
```
Comment (sijie) [inline docs/en/release_process.md]: why do we delete the previous milestone?

File: docs/en/release_process.md [hunk 24]
```diff
@@ -25,36 +37,58 @@ Eg: When creating `v0.1.1` release, will be creating
...
+
+6. Announce the release
+
+After publishing and checking the release, work with Growth to
```
Comment (sijie) [inline docs/en/release_process.md]:
```suggestion
After publishing and checking the release, work with Growth team to
```

File: docs/en/release_process.md (hunk 23 above)
Comment (zymap) [reply docs/en/release_process.md]: I think the old version has been released. There are no pull requests that need to merge to the previous milestone.  So the previous milestone is useless, right?

Comment (zymap) [reply docs/en/release_process.md]: Ah. It should be close, not delete. Will fix

**Comment by zymap** [comment]
@sijie I address your commons. PTAL. Thanks

**Pull request merged**